packages/graphql-migrations/src/abstract/getColumnTypeFromScalar.ts

```diff
@@ -117,6 +141,14 @@ export default function(
     }
   }
 
+  // graphback json
+  if (scalarType && (scalarType.name === 'GraphbackJSON' || scalarType.name === 'GraphbackJSONObject')) {
```

Comment:
Wondering if we could also check if the scalar is `JSON` or `JSONObject` here and return type json, in the event that the user is already using these scalars.

Reply:
hmm I'd say since we are not breaking anything here, if `JSON` or `JSONBObject` were there, they'd continue to work the same way. Via `@db(type)` I suppose.